Doc/c-api/arg.rst

```diff
@@ -229,12 +229,24 @@ There are three ways strings and buffers can be converted to C:
 Numbers
 -------
 
+These formats allow to represent Python numbers or single characters as C numbers.
+Formats that require :class:`int`, :class:`float` or :class:`complex` can
+also use the corresponding special methods :meth:`~object.__index__`,
+:meth:`~object.__float__` or :meth:`~object.__complex__` to convert
+the Python object to the required type.
+
+For signed integer formats, :exc:`OverflowError` is raised if the value
+is out of range for the C type.
+For unsigned integer formats, no proper range checking is done --- the
```

Comment:
```suggestion
For unsigned integer formats, no range checking is done --- the
```
What's "proper" depends on the use case.
(IMO, in the deleted paragraph below, both “It is possible to pass "long" integers” and “proper” are an artifact of the long-gone distinction between `int` & `long` Python types.)